src/dune_rules/foreign.mli

```diff
@@ -202,5 +227,5 @@ module Objects : sig
 
   val decode : t Dune_lang.Decoder.t
 
-  val build_paths : t -> ext_obj:string -> dir:Path.Build.t -> Path.Build.t list
+  val build_paths : t -> ext_obj:string -> dir:Path.Build.t -> Path.t list
```

Comment:
Why was this type changed?

Reply:
I had to modify consumers of the output of this function for my changes and noticed that all its usages where followed by a `List.map ~f:Path.build` so I upstreamed it.